lib/active_record/connection_adapters/amazon_timestream_adapter.rb

```diff
@@ -14,9 +14,8 @@ def self.amazon_timestream_connection(config)
       config = config.symbolize_keys
 
       raise ArgumentError, 'No database specified. Missing argument: database.' unless config.key?(:database)
-
-      credentials = Aws::Credentials.new config[:username], config[:password] if config[:username] && config[:password]
-      connection = Aws::TimestreamQuery::Client.new({ credentials: credentials }.compact)
+      credentials = Aws::Credentials.new config[:username], config[:password], config[:session_token] if config[:username] && config[:password]
+      connection = Aws::TimestreamQuery::Client.new({ credentials: credentials}.compact)
```

Comment:
Can you please return the space after `credentials: credentials` so it reads `credentials: credentials }` instead of `credentials: credentials}`?